src/app/header/Header.tsx

```diff
@@ -1,4 +1,4 @@
-import { Center, Grid, GridItem, Image, Button, Flex } from '@chakra-ui/react';
+import { Center, Grid, GridItem, Image, Button, Flex, Text, LinkBox } from '@chakra-ui/react';
```

Comment:
`Image` and `Button` aren't used

Reply:
Maybe we should finally setup pre-commit hooks :eyes:

Reply:
yeah, and make the import linting an error instead of a warning, it's def the most common lol